mobile/apps/partner_profile/test/templates.coffee

```diff
@@ -9,6 +9,20 @@ Partner = require '../../../models/partner'
 Artists = require '../../../collections/artists'
 fixtures = require '../../../test/helpers/fixtures'
 
+INSTITUTION        = 'institution'
+GALLERY_DEFAULT    = 'gallery_default'
+GALLERY_DEPRECATED = 'gallery_deprecated'
+GALLERY_ONE        = 'gallery_one'
+GALLERY_TWO        = 'gallery_two'
+GALLERY_THREE      = 'gallery_three'
+
+PARTNER_TYPES = [
+  INSTITUTION
+  GALLERY_ONE
+  GALLERY_TWO
+  GALLERY_THREE
+]
```

Comment:
Yea, that could work. This feels like something that can be added to `sharify` also (esp ifs going to be reused even more places).